app/view_models/nsm/v1/additional_fee.rb

```diff
@@ -39,7 +39,7 @@ def table_fields
         [
           I18n.t("nsm.additional_fees.index.#{type}"),
           format(claimed_total_exc_vat),
-          format(assessed_total_inc_vat.positive? ? assessed_total_inc_vat : nil),
+          format(any_adjustments? ? assessed_total_exc_vat : nil),
```

Comment:
Possibly worth adding a comment pointing out that for `any_adjustments?` to work we will have to set the `adjustment_comment` attribute on this class when instantiating it in `AdditionalFeesSummary`